tests/contrib/mcp/test_mcp_llmobs.py

```diff
@@ -16,9 +16,15 @@ def _assert_distributed_trace(mock_tracer, llmobs_events, expected_tool_name):
 
     all_spans = [span for trace in traces for span in trace]
     client_spans = [span for span in all_spans if span.resource == "client_tool_call"]
-    server_spans = [span for span in all_spans if span.resource == "server_tool_call"]
+    server_spans = [
+        span for span in all_spans if span.name == "mcp.tools/call" and span.resource == "request_responder"
```

Comment:
yeah seeing it here, this is a breaking change for the resource name, let's keep it as `server_tool_call`